### DISCUSSION TITLE
Qounty id segítség

### DISCUSSION THREAD

**Discussion by kornelremete** [amaximus/met_alerts_hu#11] [Q&A]
Szia!

Meg tudnád adni a qounty listát, hogy pontosan melyik megye melyik szám? Konkrétan Hajdu Bihar megye érdekelne.

Köszönöm
Kornél

**Comment by amaximus** [comment]
Az alábbi oldalon a megyére kattintva találod a county_id-t.

https://www.met.hu/idojaras/veszelyjelzes/index.php?c=b

**Comment by amaximus** [comment]
Tkp. ezen a linken tudod ellenőrizni:

https://www.met.hu/idojaras/veszelyjelzes/hover.php?id=wbhx&kod=<szám>

pl. Komárom-Esztergom megye a 11-es:
https://www.met.hu/idojaras/veszelyjelzes/hover.php?id=wbhx&kod=11
